generator.go

```diff
@@ -39,7 +39,7 @@ import (
 // UUID epoch (October 15, 1582) and Unix epoch (January 1, 1970).
 const epochStart = 122192928000000000
 
-type epochFunc func() time.Time
+type EpochFunc func() time.Time
```

Comment:
Now that this is being exported, could you please add a GoDoc-compatible comment that explains what this is:

```Go
// EpochFunc is ...
```